Gamer.py

```diff
@@ -0,0 +1,22 @@
+class Gamer:
+    def __init__(self, username='username', score=0):
+        self._username = username
+        self.score = score
+
+    def get_username(self):
+        return self._username
+
+    def set_username(self, username):
+        self._username = username
+
+    def change_score(self, other):
+        self.set_score(self.get_score() + other)
+
+    def get_score(self):
+        return int(self.score)
+
+    def set_score(self, score=0):
+        self.score = int(score)
+
+    def __str__(self):
+        return 'username = {0}\nscore = {1}\n'.format(self._username, self.score)

```

Comment:
В python 3.7 появились форматные строки:
```suggestion
        return f'username = {self._username}\nscore = {self.score}\n'
```

Reply:
Сделано